Brands API Reference update for removePoweredByOkta field addition (#2596)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/brands/index.md

}
```

> The following example resets a privacy policy URL to use the Okta default privacy policy URL and remove Powered by Okta from all pages.

Reviewer: ```suggestion
The following example resets a privacy policy URL to use the Okta default privacy policy URL and removes the Powered by Okta label from all pages.
```

| ----------------------------- | ----------------------- | --------------------------------------------------------------------------------------------------- |
| `agreeToCustomPrivacyPolicy`  | boolean                 | (Optional) Consent for updating the custom privacy policy URL. Not required when resetting the URL. |
| `customPrivacyPolicyUrl`      | String                  | Custom privacy policy URL. Default value: `null`.                                                   |

> | `removePoweredByOkta`         | boolean                 | (Optional) Flag to remove "Powered by Okta" from the experience. Defaulted to `false`               |

Reviewer: ```suggestion
| `removePoweredByOkta`         | boolean                 | (Optional) Flag that removes the "Powered by Okta" wording from the experience. Defaults to `false`               |
```

> | `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard.. Defaulted to `false`        |

Reviewer: ```suggestion
| `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. Default value: `false` .       |
```

> | `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. Default value: `false` .       |

Reviewer: ```suggestion
| `removePoweredByOkta`         | boolean                 | (Optional) Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. Default value: `false`.       |
```

}
```

> Updates setting to remove Powered by Okta

Reviewer: ```suggestion
Updates setting to remove the Powered by Okta wording
```

```json
{
  "agreeToCustomPrivacyPolicy": true,
  "removePoweredByOkta": true,
  "customPrivacyPolicyUrl": "https://www.someHost.com/privacy-policy"
}
```

### Brand Response object

The Brand Response object defines the following properties:

| Property                  | Type                    | Description                                             |
| ------------------------- | ----------------------- | ------------------------------------------------------- |
| `customPrivacyPolicyUrl`  | String                  | Custom privacy policy URL                               |

> | `removePoweredByOkta`     | boolean                 | Setting to remove "Powered by Okta" from the experience |

Reviewer: ```suggestion
| `removePoweredByOkta`     | boolean                 | Setting to remove the  "Powered by Okta" wording from the experience |
```

> | `removePoweredByOkta`     | boolean                 | Remove "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |

Reviewer: ```suggestion
| `removePoweredByOkta`     | boolean                 | Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |
```

> | `removePoweredByOkta`     | boolean                 | Removes "Powered by Okta" from the Okta-hosted sign-in page, and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |

Reviewer: ```suggestion
| `removePoweredByOkta`     | boolean                 | Removes "Powered by Okta" from the Okta-hosted sign-in page and "© 2021 Okta, Inc." from the Okta End-User Dashboard. |
```
